gpkit/constraints/sigeq.py

```diff
@@ -0,0 +1,15 @@
+"Implements SignomialEquality"
+from .set import ConstraintSet
+from ..nomials import SingleSignomialEquality
+from ..nomials.array import array_constraint
+
```

Comment:
Looking at it again, it doesn't seem that awkward? We could abstract some of it away as boilerplate, but I'm content to let this stand until we have more constraints for which we've needed to implement this kind of array/scalar input.